app/views/repositories/empty.html.erb

```diff
@@ -33,6 +33,7 @@ See doc/COPYRIGHT.rdoc for more details.
              message: l('repositories.errors.empty_repository')) %></p>
   </div>
 </div>
+<%= render partial: 'checkout_instructions', locals: { repository: @repository, instructions: @instructions } %>
```

Comment:
You are right. Since we are displaying it as a warning I agree.

We could discuss whether the warning is apropriate here, though. Since everything is fine, except for the repository being empty, which is the default for the repository of a new project.

We do not say

> :warning: There are no work packages yet.

do we?